bazaar/README.md

```diff
@@ -41,14 +41,89 @@ optional:
 * since: allows to filter by updated block height to query fresh entries only
 
 
-### storage
+#### Use Cases
+
+1. Business Management   
+   * create business
+   * edit business
+   * delete business
+2. Offering Management
+   * create offering
+   * edit offering
+   * delete offering
+3. View (RPC)
+   * businesses per community
+   * articles per community
```

Comment:
I would consistently use offerings as names instead of articles. All storage fields in a pallet are implicitly exposed via RPC and will therefore follow the storage naming anyhow.